Community/Tdarr_Plugin_TD01_TOAD_Plex_Autoscan.js

```diff
@@ -0,0 +1,91 @@
+module.exports.dependencies = [
+  'request',
+];
+
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
+module.exports.details = function details() {
+  return {
+    id: 'Tdarr_Plugin_TD01_TOAD_Plex_Autoscan',
+    Stage: 'Post-processing',
+    Name: 'Trigger Plex_Autoscan.',
+    Type: 'Video',
+    Operation: '',
+    Description: `Connects to plex_autoscan and triggers a manual search within the files directory.\n\n
+    Works with the hotio autoscan docker that can be found here https://hotio.dev/containers/autoscan/ which \n\n
+    is based on https://github.com/cloudbox/autoscan`,
+    Version: '1.0',
+    Link: '',
```

Comment:
Please enter the link to the file in the community folder.

Reply:
completed